python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,29 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+# Maximum size of a string column is 2gb
+STRINGS_UDF_DEFAULT_HEAP_SIZE = int(2e9)
```

Comment:
This doesn't match the maximum `size_type`. It is slightly short. Let's avoid problems arising from that.
```suggestion
STRINGS_UDF_DEFAULT_HEAP_SIZE = int(2 * 1024**3)
```

Reply:
Or better yet:
```suggestion
STRINGS_UDF_DEFAULT_HEAP_SIZE = 2**31
```